nestjs/src/courses/cross-checks/dto/cross-check-criteria-data.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsIn, IsNumber, IsOptional, IsString } from 'class-validator';
+
+export type CrossCheckCriteriaType = 'title' | 'subtask' | 'penalty';
```

Comment:
probably makes sense to create real enum and use it here & on line 20

Reply:
done